bin/verify-exercises

```diff
@@ -40,7 +39,11 @@ if [[ -d "./exercises/concept" ]]; then
     for concept_exercise_dir in ./exercises/concept/*; do
         if [[ -d $concept_exercise_dir ]]; then
             slug="${practice_exercise_dir##*/}"
-            run_test "$concept_exercise_dir"
+            if run_test "$concept_exercise_dir"; then
+                echo "✅ $slug"
+            else
+                echo "❌ $slug"
```

Comment:
This is a behavior change; failed tests no longer exit the script and the `verify-exercises` script will never fail tests :)